apps/native-component-list/src/components/FunctionDemo/FunctionDemo.tsx

```diff
@@ -48,6 +48,10 @@ type Props = {
    * Rendering function to render some additional components based on the function's result.
    */
   renderAdditionalResult?: (result: unknown) => JSX.Element | void;
+  /**
+   * If true, the result box will de displayed even if action returns `undefined`.
+   */
+  displayUndefinedResult?: boolean;
```

Comment:
I don't think that's necessary 🤔 
I would display `undefined` in every case as it's part of the function API I guess 😁 
```suggestion
```

Reply:
reverted